clientv3/op.go

```diff
@@ -145,10 +157,22 @@ func WithPrefix() OpOption {
 		op.end = []byte{0}
 	}
 }
+
+// WithRange specifies the range of 'Get' or 'Delete' requests.
+// For example, 'Get' requests with 'WithRange(end)' returns
+// the keys in the range [key, end).
 func WithRange(endKey string) OpOption {
 	return func(op *Op) { op.end = []byte(endKey) }
 }
+
+// WithFromKey specifies the range of 'Get' or 'Delete' requests
```

Comment:
WithFromKey specifies the range of 'Get' or 'Delete' requests to be equal and greater than they key in the argument.